conman/routes/tests/test_models.py

```diff
@@ -45,6 +49,16 @@ def test_unique_url(self):
             Route.objects.create(url='/')
 
 
+class RouteCheckTest(TestCase):
+    """Test Route.check()."""
+    def test_defer_to_handler(self):
+        """Route.check returns the result of Route.handler_class().check()."""
+        with mock.patch('conman.routes.models.Route.handler_class') as handler:
+            errors = Route.check()
+
+        self.assertEqual(errors, handler.check(Route))
```

Comment:
Aha. I don't think would be ideal in this instance, as `check` is a `classmethod` on `handler`, so `handler` itself shouldn't be called.

The assert as it stands is intended to show that the `errors` list returned from `Route.check` is delegated to `handler.check`, and that `Route` is passed in when it's called.